charts/nri-prometheus/static/lowdatamodedefaults.yaml

```diff
@@ -0,0 +1,124 @@
+# When standalone is set to false nri-prometheus requires an infrastructure agent to work and send data.
+# Default: true
+# standalone: true
+
+# Set the Metrics API url. It should not be changed unless required (for example to use FedRamp approved endpoints)
+# metric_api_url: <url>
+
+# How often the integration should run.
+# Default: "30s"
+# scrape_duration: "30s"
+
+# The HTTP client timeout when fetching data from targets.
+# Default: "5s"
+# scrape_timeout: "5s"
+
+# scrape_services Allows to enable scraping the service and not the endpoints behind.
+# When endpoints are scraped this is no longer needed
+scrape_services: true
+
+# scrape_endpoints Allows to enable scraping directly endpoints instead of services as prometheus service natively does.
+# Please notice that depending on the number of endpoints behind a service the load can increase considerably
+scrape_endpoints: false
+
+# How old must the entries used for calculating the counters delta be
+# before the telemetry emitter expires them.
+# Default: "5m"
+# telemetry_emitter_delta_expiration_age: "5m"
+
+# How often must the telemetry emitter check for expired delta entries.
+# Default: "5m"
+# telemetry_emitter_delta_expiration_check_interval: "5m"
+
+# Whether the integration should run in verbose mode or not.
+# Default: false
+verbose: false
```

Comment:
If I am not wrong customers setting lowData=true will not be able to change these defaults parameters, is it the expected behaviour?

Reply:
I modified the PR to overwrite only the transformation list